e2e/test/provisioning/ProvisioningCertificateValidationE2ETest.cs

```diff
@@ -52,61 +53,61 @@ public async Task ProvisioningServiceClient_QueryInvalidServiceCertificateHttp_F
         public async Task ProvisioningDeviceClient_RegisterAsyncInvalidServiceCertificateAmqpTcp_Fails()
         {
             using var transport = new ProvisioningTransportHandlerAmqp(ProvisioningClientTransportProtocol.Tcp);
-            ProvisioningTransportException exception = await Assert.ThrowsExceptionAsync<ProvisioningTransportException>(
-                () => TestInvalidServiceCertificate(transport)).ConfigureAwait(false);
+            Func<Task> act = async () => await TestInvalidServiceCertificate(transport);
 
-            Assert.IsInstanceOfType(exception.InnerException, typeof(AuthenticationException));
+            var error = await act.Should().ThrowAsync<DeviceProvisioningClientException>().ConfigureAwait(false);
```

Comment:
I'm new to our fluent assertions, but is there a reason we're using `var` instead of an explicit type here? Would the actual type just be too verbose and hard to read?

Reply:
Yes. In other words, it's not necessary to specify the exception type again as we already include it with `ThrowAsync<>`.